### DISCUSSION TITLE
in cms-wordpress template i face to this error  : Error: Invalid src prop [(https://criteria-group.com/wp-content/uploads/2023/05/Black-Flower-We-Are-Open-Signs-Poster-scaled.jpg)](https://criteria-group.com/wp-content/uploads/2023/05/Black-Flower-We-Are-Open-Signs-Poster-scaled.jpg) on `next/image`, hostname "criteria-group.com" is not configured under images in your `next.config.js`

### DISCUSSION THREAD

**Discussion by rahi62** [vercel/next.js#51480] [Show and tell]
in cms-wordpress template i face to this error
Error: Invalid src prop [(https://criteria-group.com/wp-content/uploads/2023/05/Black-Flower-We-Are-Open-Signs-Poster-scaled.jpg)](https://criteria-group.com/wp-content/uploads/2023/05/Black-Flower-We-Are-Open-Signs-Poster-scaled.jpg) on `next/image`, hostname "criteria-group.com" is not configured under images in your `next.config.js`

//next.config.js
if (!process.env.WORDPRESS_API_URL) {
  throw new Error(`
    Please provide a valid WordPress instance URL.
    Add to your environment variables WORDPRESS_API_URL.
  `)
}

/** @type {import('next').NextConfig} */
module.exports = {
  images: {
    domains: [
      process.env.WORDPRESS_API_URL.match(/(?!(w+)\.)\w*(?:\w+\.)+\w+/)[0], // Valid WP Image domain.
      '0.gravatar.com',
      '1.gravatar.com',
      '2.gravatar.com',
      'secure.gravatar.com',
    ],
  },
}

what is wrong?

**Comment by SamKomesarook** [comment]
You are referancing a remote asset at a URL that hasn't been configured. You need to add it to your `next.config.js` file, something like this:

```typescript
/** @type {import('next').NextConfig} */
const nextConfig = {
  images: {
    remotePatterns: [
      {
        protocol: "https",
        hostname: "criteria-group.com",
        pathname: "*",
      }
    ],
  },
}

```

Refer to the [docs](https://nextjs.org/docs/app/api-reference/components/image#remotepatterns) for further info.
